### PR TITLE
forgot password

### LINKED ISSUE #8: Forgot Password
Workflow:
user clicks on forgot password button
user enters in email
our service sends email to user which contains a link to change their password
password is changed :)

### PR DESCRIPTION
**Pull request by ctc-devops**
Workflow:
user clicks on forgot password button
user enters in email
our service sends email to user which contains a link to change their password
password is changed :)
closes #8

### CODE REVIEW

File: src/utils/auth_utils.js [hunk 1]
```diff
@@ -106,8 +107,17 @@ const registerWithEmailAndPassword = async (
...
+ * Sends a password reset email given an email
+ * @param {string} email The email to resend password to
+ */
+const sendPasswordReset = async email => {
+  try {
+    await sendPasswordResetEmail(auth, email);
+    console.log('Password Reset email sent!');
+  } catch (err) {
+    console.log(err.message);
+  }
```
Comment (Vakhshoori101) [inline src/utils/auth_utils.js]: like we talked about, no need for this try/catch

File: src/App.jsx [hunk 2]
```diff
@@ -15,6 +16,7 @@ function App() {
           <Route exact path="/" element={<Login />} />
           <Route exact path="/logout" element={<Logout />} />
           <Route exact path="/register" element={<Register />} />
+          <Route exact path="/forgotpassword" element={<ForgotPassword />} />
```
Comment (Vakhshoori101) [inline src/App.jsx:23]: sort alphabetically XD

**Review by Vakhshoori101 — CHANGES_REQUESTED** [review]
address comments

File: src/App.jsx (hunk 2 above)
Comment (nguyenethan01) [reply src/App.jsx:23]: no

**Pull request merged**